packages/medusa/src/models/payment.ts

```diff
@@ -29,23 +30,23 @@ export class Payment extends BaseEntity {
 
   @OneToOne(() => Swap)
   @JoinColumn({ name: "swap_id" })
-  swap: Swap
+  swap: Relation<Swap>
 
   @Index()
   @Column({ nullable: true })
   cart_id: string
 
   @ManyToOne(() => Cart)
   @JoinColumn({ name: "cart_id" })
-  cart: Cart
+  cart: Relation<Cart>
 
   @Index()
   @Column({ nullable: true })
   order_id: string
 
   @ManyToOne(() => Order, (order) => order.payments)
   @JoinColumn({ name: "order_id" })
-  order: Order
+  order: Relation<Order>
```

Comment:
Were these changes intended (since the title says just moving tests to `swc jest`?

Reply:
Ah nevermind saw the comments above just now 👍

Reply:
does that have any impact on manipulating a relation or its type?

Reply:
ahah nevermind `export type Relation<T> = T;`